make/go.mk

```diff
@@ -26,6 +26,8 @@ NSTEMPLATES_BASEDIR = deploy/templates/nstemplatetiers
 NSTEMPLATES_FILES = $(wildcard $(NSTEMPLATES_BASEDIR)/**/*.yaml)
 NSTEMPLATES_TEST_BASEDIR = test/templates/nstemplatetiers
 NSTEMPLATES_TEST_SUBDIRS = $(wildcard $(NSTEMPLATES_TEST_BASEDIR)/**/*)
+NOTIFICATION_BASEDIR = deploy/templates/notificationtemplates
+NOTIFICATION_FILES = $(wildcard $(NOTIFICATION_BASEDIR)/**/*.yaml)
```

Comment:
Where is it used?

Reply:
Sorry, i thought initially i might need it and then forgot to remove it =/ - removed here: cdff7a572385d6040abdf5b1a1cb55e2a987511a